src/BlazorWebView/src/Maui/Android/BlazorWebViewHandler.Android.cs

```diff
@@ -18,8 +19,26 @@ public partial class BlazorWebViewHandler : ViewHandler<IBlazorWebView, AWebView
 		private AndroidWebKitWebViewManager? _webviewManager;
 		internal AndroidWebKitWebViewManager? WebviewManager => _webviewManager;
 
+		private bool _loggerCreated;
+		private ILogger? _logger;
+
+		internal ILogger? Logger
+		{
+			get
+			{
+				if (!_loggerCreated)
+				{
+					_logger = Services!.GetService<ILogger<BlazorWebViewHandler>>();
+					_loggerCreated = true;
+				}
+				return _logger;
+			}
+		}
+
```

Comment:
Is there a reason why we are creating this lazily?

Am I correct to assume that we believe `Logging` is not a hard dependency? (Inferred from the `GetService` call).

Would it make sense to add it as a dependency instead?

Reply:
You can avoid this by doing something like `_logger ??= Services!.GetService<ILogger<BlazorWebViewHandler>>() ?? NullLogger<BlazorWebViewHandler>.Instance;

So
```csharp
		internal ILogger? Logger => _logger ??= Services!.GetService<ILogger<BlazorWebViewHandler>>() ?? NullLogger<BlazorWebViewHandler>.Instance;
```

Reply:
You know what? I think I erroneously assumed that some of these were public ctors on public types and wanted to avoid a breaking change. But now that I look again it seems they are at least mostly internal, so I'll just change the ctors to add the parameters. And if I need to get it manually I'll use the NullLogger as you suggest!